### DISCUSSION TITLE
OpenZFS Foundation

### DISCUSSION THREAD

**Discussion by bghira** [openzfs/zfs#12136] [Show and tell]
Hi all,

I'm working on registering the OpenZFS Foundation here, and wanted to know if anyone were interested in being on the board of directors, we'd be able to use this organisation to raise funds and handle executing the OpenZFS conferences.

Let me know, you can reach out here or via `foundation AT OpenZFS dot INFO`.

**Comment by ahrens** [comment]
@bghira I wasn't aware of this effort.  There is currently no standalone OpenZFS Foundation, but OpenZFS is an [associated project of SPI](https://www.spi-inc.org/projects/openzfs/), which holds the project's assets, including funds for the annual [OpenZFS Developer Summit](https://openzfs.org/wiki/OpenZFS_Developer_Summit_2020).

If you have ideas on what a separate Foundation could accomplish, and would like to help set it up, let's chat.  Perhaps we could discuss it at the next [OpenZFS Leadership Meeting](https://openzfs.org/wiki/OpenZFS_Leadership_Meeting), or we can set up a call to discuss.

**Comment by proailurus** [comment]
> handle executing the OpenZFS conferences

Sounds awfully similar to what Freenode Ltd. wanted to do.
Are you trying to take over OpenZFS as well?

https://github.com/freenode/web-7.0/commit/1194a3e71a427a669ccdddee22006416d46eeb43

**Comment by justjanne** [comment]
Either this is a mediocre-executed false flag (usually LTM reaches out differently, and communicates differently, and uses different nameservers for their parked domains), or this is real and intended as yet another semi-takeover of an open source project.

As someone who was approached by LTM in the past, I can't recommend accepting their terms.

Should this really be you, Chris Phanes, please contact me at justJanne on libera or freenode to have a short chat about your actions.

**Reply by bghira** [reply]
i don't know why people are assuming i'm Chris. I'm a different person. please stop linking this back to Freenode IRC drama. it's completely irrelevant.

**Reply by justjanne** [reply]
In that case, I wish you much success. But you should probably try to change your nick, it’s just going to hinder you going forward :(

**Reply by cmpunches** [reply]
Hi I'm the Bagira you think you're talking to.

This cool cat incarnation is not me.

He is still a very cool and intelligent dude, though, so please be nice to him.

**Reply by cmpunches** [reply]
*Another case of otherwise intelligent people on the internet making dramatic decisions without anything resembling the necessary information to justify it.  So odd to see such a tragic epistemological failure paired to such a strong attempt to imply credibility.
